.textlintrc.yml

```diff
@@ -136,6 +136,7 @@ rules:
         - '(?<![.-])java\b(?![.])'
         - Java
       - ['java[- ]?agent', Java agent]
+      - ['java ?doc(s)?', 'Javadoc$1']
```

Comment:
@svrnm @theletterf - what do you think about this rule?

Reply:
Would it catch "Java documentation" as well? So, if I write "to learn more about this, read the "Java documentation". Otherwise I am fine, since `Javadoc` seems to be the official notation